src/rpc/server/connection.zig

```diff
@@ -0,0 +1,264 @@
+const builtin = @import("builtin");
+const std = @import("std");
+
+pub const GetSockNameError = std.posix.GetSockNameError;
+pub fn getSockName(
```

Comment:
can we get some space up in here

Reply:
706f136b75d03abed6a66023acce4c00a76e85c2